### PR TITLE
fix custom property render rows issue

### LINKED ISSUE #15179: Custom Properties data for tables are only shown limited
**Affected module**
Affected module: UI

**Describe the bug**
We cannot see on UI all the custom properties of the tables (only five are visible) , clicking an "See all" has no effect
**To Reproduce**

Screenshots of custom properties::
image

**Expected behavior**
Clicking on link "VIew all" should show all Custom Properties of the object. 

**Version:**
 - OpenMetadata 1.3.0

### PR DESCRIPTION
**Pull request by harsh-vador**
### Describe your changes:

Fixes #15179 

I worked on fixing the custom properties table issue which showed only 5 properties even though it had many custom properties.

https://github.com/open-metadata/OpenMetadata/assets/58542468/2468612a-ca7d-4b53-9e4c-8916fdf43478

#
### Type of change:

- [x] Bug fix

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

**Comment by Ashish8689** [comment]
@harsh-vador  can you add a test around the same!

**Comment by harsh-vador** [comment]
> @harsh-vador can you add a test around the same!



> @harsh-vador can you add a test around the same!

Its cypress is already written

File: openmetadata-ui/src/main/resources/ui/src/components/common/CustomPropertyTable/CustomPropertyTable.tsx [hunk 1]
```diff
@@ -252,7 +252,8 @@ export const CustomPropertyTable = <T extends ExtentionEntitiesKeys>({
             <Typography.Text className="right-panel-label">
               {t('label.custom-property-plural')}
             </Typography.Text>
-            {(entityTypeDetail.customProperties ?? []).length >= maxDataCap &&
+            {maxDataCap &&
+              (entityTypeDetail.customProperties ?? []).length >= maxDataCap &&
               entityDetails.fullyQualifiedName && (
```
Comment (aniketkatkar97) [inline openmetadata-ui/src/main/resources/ui/src/components/common/CustomPropertyTable/CustomPropertyTable.tsx:257]: Can we bring this condition out of the return statement and make it more readable?

**Review by aniketkatkar97 — COMMENTED** [review]
One comment

File: openmetadata-ui/src/main/resources/ui/src/components/common/CustomPropertyTable/CustomPropertyTable.tsx [hunk 2]
```diff
@@ -189,6 +189,34 @@ export const CustomPropertyTable = <T extends ExtentionEntitiesKeys>({
     onExtensionUpdate,
   ]);
 
+  const renderViewAll = useMemo(() => {
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/common/CustomPropertyTable/CustomPropertyTable.tsx:192]:
```suggestion
  const viewAllBtn = useMemo(() => {
```

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**